package.json

```
@@ -6,6 +6,9 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
+    "bootstrap": "^4.6.0",
```

`4.6.0` でバグが出ていますので，

```suggestion
    "bootstrap": "^4.5.1",
```

に変更して

```
yarn install --check-files
```

を実行して再プッシュして下さい。